catboost/libs/model/eval_processing.h

```diff
@@ -114,22 +123,22 @@ namespace NCB::NModelEvaluation {
     class TEvalResultProcessor {
     public:
         TEvalResultProcessor(
-            size_t docCount,
-            TArrayRef<double> results,
-            EPredictionType predictionType,
-            TScaleAndBias scaleAndBias,
-            ui32 approxDimension,
-            ui32 blockSize,
-            TMaybe<double> binclassProbabilityBorder = Nothing()
+                size_t docCount,
+                TArrayRef<double> results,
+                EPredictionType predictionType,
+                TScaleAndBias scaleAndBias,
+                ui32 approxDimension,
+                ui32 blockSize,
+                TMaybe<double> binclassProbabilityBorder = Nothing()
         );
 
         inline TArrayRef<double> GetResultBlockView(ui32 blockId, ui32 dimension) {
             return Results.Slice(
-                blockId * BlockSize * dimension,
-                Min<ui32>(
-                    BlockSize * dimension,
-                    Results.size() - (blockId * BlockSize * dimension)
-                )
+                    blockId * BlockSize * dimension,
+                    Min<ui32>(
+                            BlockSize * dimension,
+                            Results.size() - (blockId * BlockSize * dimension)
+                    )
```

Comment:
Return old indents please)
And in other places too.

Reply:
Ok (actually CLion changed them without my consent )